pkg/api/grpc/daprinternal.go

```diff
@@ -312,6 +312,25 @@ func (a *api) CallActorReminder(ctx context.Context, in *internalv1pb.Reminder)
 		Period:    in.GetPeriod(),
 		TTL:       in.GetExpirationTime().String(),
 	})
+	if errors.Is(err, actors.ErrReminderCanceled) {
```

Comment:
I haven't looked at the code flow closely, but shouldn't this error be handled internally in the actor runtime instead of here?

Reply:
oh yes, I added it to the `ExecuteLocalOrRemoteActorReminder` so this check can be removed.